spec/models/application_draft_spec.rb

```diff
@@ -57,15 +57,16 @@ def build_draft
       it_behaves_like 'proxies :apply validation', :heard_about_it
       it_behaves_like 'proxies :apply validation', :working_together
       it_behaves_like 'proxies :apply validation', :why_selected_project1
+      it_behaves_like 'proxies :apply validation', :work_weeks
 
-      context 'required fields for voluntary mode' do
-        it { is_expected.not_to validate_presence_of :voluntary_hours_per_week }
-        it { is_expected.not_to validate_presence_of(:voluntary_hours_per_week).on(:apply) }
+      context 'required fields for deprecated_voluntary mode' do
```

Comment:
Please remove all the logic and tests related to the deprecated columns _(in the whole file)_